examples/eSealing/src/eSealing.tsx

```diff
@@ -0,0 +1,413 @@
+/* eslint-disable no-alert */
+
+import React, { useEffect, useState, useRef, ChangeEvent } from 'react';
+import Switch from 'react-switch';
+import {
+    toBuffer,
+    deserializeReceiveReturnValue,
+    serializeUpdateContractParameters,
+    JsonRpcClient,
+} from '@concordium/web-sdk';
+import sha256 from 'sha256';
+import { version } from '../package.json';
+
+import { register } from './utils';
+import {
+    E_SEALING_CONTRACT_NAME,
+    E_SEALING_CONTRACT_INDEX,
+    E_SEALING_RAW_SCHEMA,
+    CONTRACT_SUB_INDEX,
+} from './constants';
+
+import { withJsonRpcClient } from './wallet/WalletConnection';
+import { WalletConnectionProps } from './wallet/WithWalletConnection';
+import { WalletConnectionTypeButton } from './WalletConnectorTypeButton';
+
+const blackCardStyle = {
+    backgroundColor: 'black',
+    color: 'white',
+    width: '500px',
+    borderRadius: 10,
+    margin: '10px 0px 10px 0px',
+    padding: '10px 18px',
+    border: '1px solid #308274',
+};
+
+const ButtonStyle = {
+    color: 'white',
+    borderRadius: 10,
+    margin: '7px 0px 7px 0px',
+    padding: '10px',
+    width: '100%',
+    border: '1px solid #26685D',
+    backgroundColor: '#308274',
+    cursor: 'pointer',
+    fontWeight: 300,
+    fontSize: '14px',
+};
+
+const ButtonStyleDisabled = {
+    color: 'white',
+    borderRadius: 10,
+    margin: '7px 0px 7px 0px',
+    padding: '10px',
+    width: '100%',
+    border: '1px solid #4B4A4A',
+    backgroundColor: '#979797',
+    cursor: 'pointer',
+    fontWeight: 300,
+    fontSize: '14px',
+};
+
+const InputFieldStyle = {
+    backgroundColor: '#181817',
+    color: 'white',
+    borderRadius: 10,
+    width: '100%',
+    border: '1px solid #308274',
+    margin: '7px 0px 7px 0px',
+    padding: '10px 20px',
+};
+
+async function viewFile(rpcClient: JsonRpcClient, fileHash: string) {
+    const fileHashByteArray = fileHash.split(',').map((s) => parseInt(s, 10));
+
+    const param = serializeUpdateContractParameters(
+        E_SEALING_CONTRACT_NAME,
+        'getFile',
+        fileHashByteArray,
+        toBuffer(E_SEALING_RAW_SCHEMA, 'base64')
+    );
+
+    const res = await rpcClient.invokeContract({
+        method: `${E_SEALING_CONTRACT_NAME}.getFile`,
+        contract: { index: E_SEALING_CONTRACT_INDEX, subindex: CONTRACT_SUB_INDEX },
+        parameter: param,
+    });
+
+    if (!res || res.tag === 'failure' || !res.returnValue) {
+        throw new Error(
+            `RPC call 'invokeContract' on method '${E_SEALING_CONTRACT_NAME}.view' of contract '${E_SEALING_CONTRACT_INDEX}' failed`
+        );
+    }
+
+    const returnValues = deserializeReceiveReturnValue(
+        toBuffer(res.returnValue, 'hex'),
+        toBuffer(E_SEALING_RAW_SCHEMA, 'base64'),
+        E_SEALING_CONTRACT_NAME,
+        'getFile',
+        2
+    );
+
+    if (returnValues.Some !== undefined) {
+        return returnValues.Some[0];
+    }
+    return { timestamp: 'Not registered', witness: 'Not registered' };
```

Comment:
Is it intended to have the string `Not registered` as a value of a field called `timestamp`? Instead of using `null` or some such?

Reply:
I changed it to using `null` here.